turtle.js

```diff
@@ -0,0 +1,138 @@
+// worked on it on Monday Nov 19, 2018 at 930pm
+class Turtle {
+
+    constructor (x, y){
+        this.x = x;
+        this.y = y;
+        this.face = 'east';
+        this.coordinates = [];
+        this.coordinates.push([this.x, this.y])
+        
+   }
+
+   allPoints(){
+       
+         return this.coordinates
+        
+   }
+   
+   forward(steps) {
+
+     if (this.face === 'east'){
+          for (let i = 0; i < steps; i++) {
+          this.x += 1;
+          this.coordinates.push([this.x, this.y])
+          }
+     } else if (this.face === 'south') {
+          for (let i = 0; i < steps; i++) {
+          this.y += 1;   
+          this.coordinates.push([this.x, this.y])
+          }
+     } else if (this.face === 'west') {
+          for (let i = 0; i < steps; i++) {
+          this.x += -1;
+          this.coordinates.push([this.x, this.y])
+          }
+     } else if (this.face === 'north') {
+          for (let i = 0; i < steps; i++) {
+          this.y += -1
+          this.coordinates.push([this.x, this.y])
+          }
+     }
+   }
+   right(){
+        if (this.face === 'east') {
+             this.face = 'south'
+        } else if (this.face === 'south') {
+             this.face = 'west'
+        } else if (this.face === 'west') {
+             this.face = 'north'
+        } else if (this.face === 'north') {
+             this.face = 'east'
+        }
+     }
+
+   left(){
+     if (this.face === 'east') {
```

Comment:
this would be an excellent place to use a switch statement!
```js
left() {
    switch(this.face) {
        case 'east':
            this.face = 'north';
            break;
        case 'north':
            this.face = 'west';
            break;
        case 'west':
            this.face = 'south';
            break;
        case 'south':
            this.face = 'east';
            break;
    }
}
```